notary-server/notary-server.Dockerfile.dockerignore

```diff
@@ -0,0 +1,12 @@
+# exclude everything
```

Comment:
Uses this prefix for this dockerignore file so that it will be used when the notary-server image is built; and changed to whitelisting the entire folder (e.g. `tlsn`, `components`) instead of individual files because that will be too much to specify as we are copying dependency files from core libs